examples/file-io.rs

```diff
@@ -0,0 +1,58 @@
+//! Run ruperf against some file calls
+//!
+//! This is a small program to run ruperf against. Write and read a file. This 
+//! should call `syscall_write` and `syscall_read`. 
+//! 
+//! The test environment should not need to be a create and should only include
```

Comment:
What does this mean? "should not need to be a create"?

Reply:
*crate, not create, probably